src/bin/rustc.rs

```diff
@@ -32,22 +32,22 @@ Usage:
     cargo rustc [options] [--] [<opts>...]
 
 Options:
-    -h, --help               Print this message
-    -p SPEC, --package SPEC  The profile to compile for
-    -j N, --jobs N           The number of jobs to run in parallel
-    --lib                    Build only this package's library
-    --bin NAME               Build only the specified binary
-    --example NAME           Build only the specified example
-    --test NAME              Build only the specified test target
-    --bench NAME             Build only the specified benchmark target
-    --release                Build artifacts in release mode, with optimizations
-    --features FEATURES      Features to compile for the package
-    --no-default-features    Do not compile default features for the package
-    --target TRIPLE          Target triple which compiles will be for
-    --manifest-path PATH     Path to the manifest to fetch dependencies for
-    -v, --verbose            Use verbose output
-    -q, --quiet              No output printed to stdout
-    --color WHEN             Coloring: auto, always, never
+    -h, --help                   Print this message
+    -p SPEC, --package SPEC ...  The profile to compile for
```

Comment:
This is actually a command where it's always an error to specify more than one package, so it's ok to leave this as it was.